src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/PetsPerProjectServiceAccountProvider.scala

```diff
@@ -19,12 +19,4 @@ class PetsPerProjectServiceAccountProvider(config: Config) extends SamServiceAcc
   override def getNotebookServiceAccount(userInfo: UserInfo, googleProject: GoogleProject)(implicit executionContext: ExecutionContext): Future[Option[WorkbenchEmail]] = {
     Future(None)
   }
-
-  override def listUsersStagingBucketReaders(userEmail: WorkbenchEmail)(implicit executionContext: ExecutionContext): Future[List[WorkbenchEmail]] = {
```

Comment:
Talked to Vikram: these are common to all Sam ServiceAccountProvider implementations